src/parse.js

```diff
@@ -276,6 +290,87 @@ export class CellParser extends Parser {
   }
 }
 
+export class TemplateCellParser extends CellParser {
+  constructor({tag, raw, ...options}, ...args) {
+    super(options, ...args);
+    this.tag = tag;
+    this.raw = raw;
+  }
+  parse() {
+    return this.parseCell(this.startNode());
+  }
```

Comment:
I don’t think we need this, since we should be inheriting CellParser’s parseTopLevel which calls parseCell?

```suggestion
```

Reply:
I'll look at this again when I get back to my computer, but it was necessary since `parse` calls parseTopLevel otherwise. I think either this override or an override for parseTopLevel is necessary but I'll confirm later

Reply:
If parseTopLevel calls parseCell it should be fine, since TemplateCellParser overrides CellParser’s parseCell.

Reply:
It's failing because `parse` [calls `this.nextToken()`](https://github.com/acornjs/acorn/blob/7deba41118d6384a2c498c61176b3cf434f69590/acorn/src/state.js#L88-L92). Since we do our own initialization of the parser state, I think we want to avoid that -- what do you think? Just pushed a working version with an override that is identical but omits `this.nextToken()`.

Reply:
Ah… right. I pushed a commit which overrides initialContext so that we can use parse as-is.

Reply:
Nice!